app/models/calculated_value.rb

```diff
@@ -1,3 +1,9 @@
+class String
+    def is_i?
+       /\A[-+]?\d+\z/ === self
```

Comment:
Not sure if this is necessary. `String#to_i` will return the number inside a string or zero if there is no number inside the string. Not sure that's what you want here, but I wanted to mention it just in case.